modules/api/README.md

```diff
@@ -0,0 +1,50 @@
+# API
+
+## Purpose
+
+The api module is intended to provide a stable, versioned REST interface
+for interacting with the core LORIS functionality without requiring backend
+SQL access.
+
+## Intended Users
+
+The module has 2 intended use cases:
+1. Users (or creators) of LORIS data that are external to the LORIS core
+   software that must extract or import data from LORIS.
+2. Internal LORIS modules which retrieve data from the API.
+
+## Scope
+
+The API only provides information about concepts that are cross-module
```

Comment:
concepts?  how about instead: entity/object?

Reply:
Yes, concepts is vague. Would also suggest `entity or object` instead of using a slash for readability.